src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/CreateCommitmentsView.sql

```diff
@@ -206,6 +206,7 @@ SET @VSQL4=
 	 --   , CAST(C.LegalEntityAddress as nvarchar(256))                as LegalEntityRegisteredAddress
 	      , CAST(AcctLE.Address as nvarchar(256))                as LegalEntityRegisteredAddress
 		  , ISNULL(cast(A.MadeRedundant as int),-1)                           as MadeRedundant
+		  , C.CreatedOn												  		as CreatedOn
```

Comment:
This should come from Apprenticeship Table and not commitments

Reply:
CreatedOn taken from Apprenticeship table and added to ASData_PL.DAS_Commitments_LevyInd views as well